pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -1433,12 +1433,17 @@ func (s *Service) reconcileV2LBTags(lb *infrav1.LoadBalancer, desiredTags map[st
 	return nil
 }
 
-func (s *Service) getHealthCheckELBProtocol() *infrav1.ELBProtocol {
+func (s *Service) getHealthCheckTarget() string {
 	controlPlaneELB := s.scope.ControlPlaneLoadBalancer()
+	var protocol *infrav1.ELBProtocol
+	protocol = &infrav1.ELBProtocolSSL
```

Comment:
```suggestion
	var protocol *infrav1.ELBProtocol = &infrav1.ELBProtocolSSL
```

Reply:
Why not apply this simplification @faiq?

Reply:
```
pkg/cloud/services/elb/loadbalancer.go:1438:15: ST1023: should omit type *infrav1.ELBProtocol from declaration; it will be inferred from the right-hand side (stylecheck)
	var protocol *infrav1.ELBProtocol = &infrav1.ELBProtocolSSL
```

Reply:
Heh. :DDD Thanks, I didn't know that. Interesting. Oh right, because it should have been the interface type? 🤔 ah, never mind, it's probably okay.

Reply:
totally correct. i dont know why i missed that one thank you @Skarlso i applied the changes!